CHANGELOG.rst

```diff
@@ -6,6 +6,207 @@ Changelog
+  action type.
+* [:backend:`5575`] Added support for WMS tile layers in the map component, along
+  with import/export functionality.
+* [:backend:`5479`] The eIDAS (via OIDC) configuration is now simplified - you can specify
+  which claims can contain a BSN and/or Pseudo ID.
+* [:backend:`5060`] Redis Sentinel is now supported as high availability strategy for the background
+  jobs message broker.
+* [:backend:`5419`] Added prefill plugins for the new Yivi and eIDAS authentication backends.
+* [:backend:`5515`] Yivi Attribute groups now have a system-generated unique identifier.
+* [:backend:`5515`] You can now export and import Yivi attribute groups.
+* [:backend:`5253`] The BRT (grey, pastel, water) background tile layers are now available by
+  default in an Open Forms instance.
+* [:backend:`5251`] The identifier field from the map background tile layer is now automatically
+  populated from its label.
+
+* [:backend:`4879`] Finished the support for Worldline as payment provider:
+
+    - Support for Worldline's ``variant`` and ``descriptor`` fields.
+    - The merchant reference is generated by Open Forms, similar to the Ogone plugin.
+    - Ogone merchants are automatically migrated where possible.
+    - Webhook configuration (if configured in a older patch release) is automatically migrated.
+    - Moved the location in the admin of the feedback URL's used for the Worldline
+      payment webhooks to the webhook configuration page.
+    - Added a bulk action to migrate forms from Ogone to Wordline.
+
+* [:backend:`5133`] Added a feature flag to enable the new *experimental* renderer.
+* [:backend:`3999`] Added support for Open Telemetry metrics. All available metrics and details can be
+  found in the "Observability" documentation.
+* [:backend:`5478`] Added additional Yivi documentation.
+* [:backend:`5451`, :backend:`3999`] Added Open Telemetry support. The following metrics
+  are now exposed:
+
+    - HTTP request durations.
+    - Number of "active requests".
+    - Number of users, logins, logouts, failed logins, axes lockouts.
+    - Submission starts, completions, paused and steps submitted.
+    - Total amount of submissions by form or lifecycle stage.
+    - Attachment file sizes and amount uploaded.
+
+* [:backend:`2324`] Reworked part of the logic engine in preparation of further
+  performance improvements, so that we can correctly reason about variable data types.
+* [:backend:`4879`] You can now use the "Worldline" payment provider, which is the
+  successor to Ogone legacy.
+* [:sdk:`825`] Added children component and updated email digest.
+* [:backend:`5382`] Forms now have an internal remarks field.
+* [:backend:`5268`] Added "Partners Roltype" and "Partners omschrijving" registration
+  configuration options for the ZGW APIs and StUF-ZDS registration plugins.
+* [:backend:`5428`] Updated the eIDAS (OIDC) LoA-Levels.
+
+**Bugfixes**
+
+* [:backend:`5646`] Updated the children component styling.
```

Comment:
no need to mention "bug fixes" for features that haven't been released in a stable version - it creates unnecessary panic